android/src/test/java/org/mozilla/mozstumbler/client/UpdaterTest.java

```diff
@@ -59,16 +62,14 @@ public void testUpdater() {
         ServiceLocator.getInstance().putService(IHttpUtil.class, mockHttp);
         ServiceLocator.getInstance().putService(ISystemClock.class, clock);
 
-        // wifi is always unavailable
+        // network is always available
```

Comment:
Re-reading this comment, I don't like either comment now.  

Let's make it really clear with something like "Skip the exclusive wifi-only check".